apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.html

```diff
@@ -7,7 +7,12 @@
 >
   <mat-expansion-panel-header>
     <mat-panel-title class="company-info__header" [data-qa]="'panel-title'">
-      <img [src]="logo" [alt]="title" class="company-info__logo" *ngIf="logo" />
+      <img
+        [src]="logo"
+        [alt]="'Logo for ' + title + ' deployment'"
```

Comment:
Logo is fine for now I believe. Although it probably may not be their logo, may even be just a random image... But it's representing the deployment logo.

For consistency, maybe use this instead:
````
[alt]="title + ' deployment Logo'"
````